src/anshuKJun20/PrimeNumber.java

```diff
@@ -0,0 +1,47 @@
+package anshuKJun20;
+
+public class PrimeNumber {
+  void findPrimeNo(int number) {
+	  double itsSquareno = Math.sqrt(number);
+	  
+	  for(int i=2; i<= itsSquareno; i++) {
+		  if(number%i == 0)
+		  {
+			  System.out.println(number+" is not a prime number");
+		  }
+	  }
+	  System.out.println(number+" is prime number");
+  }
+  void addPrimeNumber() {
```

Comment:
Why this separate method is required ? You should directly write count and some logic in findPrimeNo() method. Take two instance variable.